OnlineStore/src/main/kotlin/Product.kt

```diff
@@ -6,22 +6,11 @@ class Product(image:String, description: String, attribute: String, longDescript
     private var productPrice: Double = price
     private var productReference:String = reference
     private var productLongDescription:String = longDescription
-    fun showProductImage():String{
-        return productImage
-    }
-    fun showProductDescription(): String{
-        return  productDescription
-    }
-    fun showProductAttribute(): String{
-        return  productHighlightAttribute
-    }
-    fun showProductPrice(): Double{
-        return  productPrice
-    }
-    fun showProductReference(): String{
-        return  productReference
-    }
-    fun showLongDescription():String{
-        return productLongDescription
-    }
+
+    fun showProductImage(): String =  productImage
```

Comment:
All this methods are innecessary in kotlin. There's a better use of the language to avoid this ;)

Reply:
I just do it using "private set" thanks for the feedback 😄